client/src/App.vue

```diff
@@ -1190,7 +1193,12 @@
 						</div>
 						<div class="welcome-section">
 							<ul>
-								<li><a @click="displayedModal = 'importdeck'">Import deck list</a></li>
+								<li>
+									<a @click="displayedModal = 'importdeck'">Card List Importer</a>
+									(Load a deck list or card pool to tinker around with.
+									<a href="https://mtgadraft.herokuapp.com/cubeformat.html" target="_blank">Learn about the supported structure</a>
+									, you can also separate cards in "Mainboard" and "Sideboard".)
+								</li>
```

Comment:
I don't like this change.
"Card List Importer" may be a better name, but I don't think you need to explain what it does, at least not here.
We could add some information on the supported format but again, not here. Maybe succinctly describe the format directly in the modal (Most likely a reference to the Arena importer rather than the full syntax?), but not a link to the cube format as those are two different things (yes, the card lines are identical, but the overall formats are different).

Reply:
I wasn't sure either. Hence I wanted to hear your opinion.
It was the most obvious place for a quick note as I was working with the other sentences anyway. :)

>Maybe succinctly describe the format directly in the modal (Most likely a reference to the Arena importer rather than the full syntax?)

Indeed, putting the hint in the modal should be best!

How would you phrase such an "Arena importer" reference?

Reply:
Update the placeholder to "Paste list in Arena format here..."? Not sure!

Reply:
I'm not sure if it's good to assume users know exactly what the "Arena format" is.
It seems obvious and simple, but still... many users don't know about it.

Some Google search revealed [this official support article](https://mtgarena-support.wizards.com/hc/en-us/articles/360049857771), the only information by wizards  I could find. The format isn't described there either, but it looks to be super easy and straight forward... I have to admit that. :)
But it still doesn't tell the users what the format is about. The user is left alone with assumptions.

<br>

If multi line placeholder text is possible, putting a simple example there could work.
Otherwise 1-2 sentences below the import button?

Reply:
I don't except the user to know exactly what's the Arena format, just that a list designed to be consumed by Arena (from this app or any another source) should work fine.

Reply:
>a list designed to be consumed by Arena should work fine

That's maybe enough information. 👍 [`67391c6` (#193)](https://github.com/Senryoku/MTGADraft/pull/193/commits/67391c6e37505c1e12c33f0608d2d6f7c78b3f1c)